<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>896</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1074</width>
    <height>856</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1074</width>
    <height>896</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GSChat</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/GS.png</normaloff>:/images/GS.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-top-color: rgb(99, 135, 255);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>40</width>
    <height>40</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>325</width>
        <height>835</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>325</width>
        <height>835</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="topMargin">
        <number>35</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_logo">
         <property name="minimumSize">
          <size>
           <width>303</width>
           <height>293</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>303</width>
           <height>293</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="pixmap">
          <pixmap resource="resource.qrc">:/images/BG1.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>500</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>500</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>20</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton_newConversation">
               <property name="minimumSize">
                <size>
                 <width>135</width>
                 <height>33</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>135</width>
                 <height>33</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;Roman&quot;;</string>
               </property>
               <property name="text">
                <string>新建会话</string>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/images/新建会话.png</normaloff>:/images/新建会话.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_deleteConversation">
               <property name="minimumSize">
                <size>
                 <width>135</width>
                 <height>33</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>135</width>
                 <height>33</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;Roman&quot;;</string>
               </property>
               <property name="text">
                <string>删除会话</string>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/images/删除会话.png</normaloff>:/images/删除会话.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>50</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="resource.qrc">:/images/会话.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>270</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>270</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;仿宋&quot;;
font: 9pt &quot;Roman&quot;;</string>
               </property>
               <property name="text">
                <string>会话列表</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_history">
            <property name="minimumSize">
             <size>
              <width>296</width>
              <height>360</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>296</width>
              <height>360</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>725</width>
        <height>827</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>725</width>
        <height>827</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="listWidget_chat">
         <property name="minimumSize">
          <size>
           <width>698</width>
           <height>639</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>698</width>
           <height>639</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>695</width>
           <height>83</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>695</width>
           <height>83</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,15,0,0">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_model">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>65</height>
             </size>
            </property>
            <property name="text">
             <string>model</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/images/GSUltra.jpg</normaloff>:/images/GSUltra.jpg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>65</width>
              <height>65</height>
             </size>
            </property>
            <property name="popupMode">
             <enum>QToolButton::MenuButtonPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_request">
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>65</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1300</width>
              <height>65</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;仿宋&quot;;
border-color: rgb(3, 3, 3);</string>
            </property>
            <property name="placeholderText">
             <string>请在这里输入你想询问的问题哦~</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_send">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>65</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>75</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/images/send.png</normaloff>:/images/send.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGSLite">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/GSLite.png</normaloff>:/images/GSLite.png</iconset>
   </property>
   <property name="text">
    <string>GSLite</string>
   </property>
  </action>
  <action name="actionGSPro">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/GSPro.jpg</normaloff>:/images/GSPro.jpg</iconset>
   </property>
   <property name="text">
    <string>GSPro</string>
   </property>
  </action>
  <action name="actionGSMax">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/GSMax.jpg</normaloff>:/images/GSMax.jpg</iconset>
   </property>
   <property name="text">
    <string>GSMax</string>
   </property>
  </action>
  <action name="actionGSUltra">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/GSUltra.jpg</normaloff>:/images/GSUltra.jpg</iconset>
   </property>
   <property name="text">
    <string>GSUltra</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
